### PR TITLE
upgrade to Jekyll assets 3

### PR DESCRIPTION
**Pull request by DirtyF**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -17,9 +17,9 @@
   </style>
 
   {% comment %} Async load of non critical CSS {% endcomment %}
-  <link rel="preload" href="{% asset_path non-critical-styles %}" as="style" onload="this.onload=null;this.rel='stylesheet'" />
```
[the remark this answers is not in the document]
Comment (nhoizey) [reply _includes/head.html]: @DirtyF it looks like the API changed a lot.

`{% asset %}` used to inline the asset (CSS, JavaScript or SVG), but now it generates the `<link…>` or `<script>` tag… :-/

I don't understand the `@inline` "argument" in <https://github.com/envygeeks/jekyll-assets#arguments>, maybe it's the solution.

**Comment by nhoizey** [comment]
Thanks @DirtyF !

**Comment by DirtyF** [comment]
T.I.L pour la nouvelle syntaxe 👍

**Comment by nhoizey** [comment]
;-)

**Pull request merged**